coderd/agentmetrics/labels.go

```diff
@@ -0,0 +1,8 @@
+package agentmetrics
+
+const (
+	TemplateNameLabel  = "template_name"
```

Comment:
nit: sort alphabetically?

Reply:
Autocomplete is pretty good at fuzzy matching these days. That said, there is value in both approaches, I often struggle to decide which to go with 😄.